docs/setup-guide.mdx

```diff
@@ -2,53 +2,63 @@
-- [NodeJS 14+](https://nodejs.org/)
-- A code editor. We recommend [VSCode](https://code.visualstudio.com/).
-- [Rojo 6+](https://rojo.space/). The easiest way to install it is through the [Rojo VSCode plugin](https://marketplace.visualstudio.com/items?itemName=evaera.vscode-rojo).
-
-## Project Setup
-
-To start using roblox-ts to create a game project, follow these steps:
-
-<!-- Unfortunately, we need to explicitly number these steps because there are images between them :( -->
+:::caution PowerShell
+If you're using PowerShell for the first time, you may get an error about "running scripts is disabled on this system". You can fix this by running the following in PowerShell as an administrator:
 
-1. Create a new folder and give it a name. The folder name cannot contain spaces. We are going to name ours `my-project`.
+`Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser`
 
-<center><img src={require("../static/img/setup-guide/new-folder.png").default} /></center>
+[More info here](./articles/powershell.mdx)
+:::
 
-2. Open the folder using your code editor (or your command line if your code editor does not have one integrated).
+## Project Setup
 
-<center><img src={require("../static/img/setup-guide/open-with-vscode.png").default} /></center>
+In this guide, we will introduce how to create a roblox-ts game.
 
-3. Run `npm init roblox-ts` in your command line to start the interactive project setup.
+Make sure you have an up-to-date version of Node.js installed and your current working directory is the one where you intend to create a project. Run the following command in your command line:
 
-:::caution PowerShell
-If you're using PowerShell for the first time, you may get an error about "running scripts is disabled on this system". You can fix this by running the following in PowerShell as an administrator:
+```bash npm2yarn
+npm init roblox-ts
+```
 
-`Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser`
+The tool will guide you through some questions to set up your project:
+1. `Project directory` - What is the name of the folder you want to put your project in? This folder shouldn't exist yet, the tool will create it for you!
+2. `Select template` -  What type of roblox-ts project is this? roblox-ts can be used to build plugins and packages too, but in this case we want to make a game, so press `Enter` once.
+3. `Configure Git` - Do you want to use [Git](https://git-scm.com/) for version control? Press `Y` or `N` to make your choise.
+4. `Configure ESLint` - Do you want to use [ESLint](https://eslint.org/) to check your code for correctness?
+5. `Configure Prettier` - Do you want to use [Prettier](https://prettier.io/) to format your code?
+6. `Configure VSCode Project Settings` - Do you want the tool to create a `.vscode` folder with settings optimized for roblox-ts development? If you're using VSCode to write code, you should use this.
 
```

Comment:
If the user has multiple package managers installed, it will cause a step 7 for selecting which one to use. I'd suggest adding that to the list, and recommending `npm` only during the guide.